modules/ps_shoppingcart/modal.tpl

```diff
@@ -58,18 +58,18 @@
               <div class="mb-3">
                 <div class="blockcart-modal__total product-subtotal">
                   <span class="label">{l s='Subtotal:' d='Shop.Theme.Checkout'}</span>
-                  <span class="value">{$cart.subtotals.products.value}</span>
+                  <span class="subtotals value">{$cart.subtotals.products.value}</span>
                 </div>
                 {if $cart.subtotals.shipping.value}
                   <div class="blockcart-modal__total product-shipping">
                     <span class="label">{l s='Shipping:' d='Shop.Theme.Checkout'}</span>
-                    <span class="value">{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</span>
+                    <span class="shipping value">{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</span>
```

Comment:
Is the `shipping` class coming from the module? Otherwise let's use BEM like said previously